dali/pipeline/operators/expressions/arithmetic.h

```diff
@@ -128,50 +132,54 @@ struct ExprImplTask {
 };
 
 template <typename Backend>
-inline void CreateExecutionOrder(std::vector<ExprImplTask> &order, const ExprNode &expr,
-                                 ExprImplCache &cache) {
+inline void CreateExecutionTasks(std::vector<ExprImplTask> &order, const ExprNode &expr,
+                                 ExprImplCache &cache, cudaStream_t stream) {
   if (expr.GetNodeType() != NodeType::Function) {
     return;
   }
-  auto &func = dynamic_cast<const ExprFunc&>(expr);
+  auto &func = dynamic_cast<const ExprFunc &>(expr);
   for (int i = 0; i < expr.GetSubexpressionCount(); i++) {
-    CreateExecutionOrder<Backend>(order, func[i], cache);
+    CreateExecutionTasks<Backend>(order, func[i], cache, stream);
```

Comment:
Is the stream used anywhere?

Reply:
Will be in the next PR. I want to put in the context.